CSharp/BatchExplorer/ViewModels/ResizePoolViewModel.cs

```diff
@@ -72,6 +86,34 @@ public string DeallocationOptionString
                 this.deallocationOptionString = value;
                 this.FirePropertyChangedEvent("DeallocationOptionString");
             }
+        }
+
+        private bool isFixed;

```

Comment:
What's the difference `IsFixed` and `IsAutoscale` ? It seems like they can be represented by 1 boolean instead of 2?  Since if `IsAutoScale == false` then `IsFixed == true` right?